src/components/questDisplay.html

```diff
@@ -69,6 +70,22 @@
                     </td>
                 </tr>
                 <!-- /ko -->
+                <!-- ko if: $data.curQuestObject().quests -->
+                <!-- ko foreach: $data.curQuestObject().quests -->
+                <tr>
+                    <td colspan="2">
+                        <div class="progress p-0">
+                            <div class="progress-bar bg-success" role="progressbar"
+                                data-bind="attr:{ style: 'width:' + ($data.progress() * 100) + '%'}"
+                                aria-valuemin="0" aria-valuemax="100">
+                                <span data-bind="text: ko.pureComputed(function(){return $data.progressText()})">0/1000</span>
+                            </div>
+                        </div>
+                    </td>
+                </tr>
+                <!-- /ko -->
+                <!-- /ko -->
+                <!-- /ko -->
```

Comment:
Could we increase the nested indentation here just so it's easier to see what's exactly happening.